kombu/transport/pyro.py

```diff
@@ -1,42 +1,64 @@
-"""Pyro transport.
+"""Pyro transport, and Kombu Broker daemon.
 
 Requires the :mod:`Pyro4` library to be installed.
+
+To use the Pyro transport with Kombu, use an url of the form:
+``pyro://localhost/kombu.broker``
+
+The hostname is where the transport will be looking for a Pyro name server,
+which is used in turn to locate the kombu.broker Pyro service.
+This broker can be launched by simply executing this transport module directly,
+with the command: ``python -m kombu.transport.pyro``
 """
+
 from __future__ import absolute_import, unicode_literals
 
 import sys
 
-from kombu.five import reraise
-from kombu.utils.objects import cached_property
-
+from ..five import reraise, Queue, Empty
```

Comment:
Why the relative imports?

Reply:
it's what I'm used to when importing stuff from other submodules from the same library, but I now see that this is not used in kombu's code base. It's trivially changed ofcourse.

Reply:
please revert as this change isn;t necessary

Reply:
Actually, relative imports _are_ used in other places as well... for instance in librabbitmq.py, pyamqp.py etc.  Isn't this a bit arbitrary?